browser/main/lib/dataApi/createSnippet.js

```diff
@@ -0,0 +1,21 @@
+import fs from 'fs'
+import crypto from 'crypto'
+import consts from 'browser/lib/consts'
+
+function createSnippet (snippets) {
```

Comment:
why didn't you write any tests?!
Tests are good!

Reply:
oh I was busy at that time, I'll add some tests for it :smile: